src/analysis/rust_type.rs

```diff
@@ -160,11 +161,11 @@ pub fn rust_type_full(
                         ok("std::ffi::OsString")
                     }
                 }
-                Type if env.namespaces.glib_ns_id == library::MAIN_NAMESPACE => ok("types::Type"),
+                Type if env.library.is_glib_crate() => ok("crate::types::Type"),
```

Comment:
Nicer if we could go with `format!("{}::types::Type", some_helper_function(&env.library))` here and elsewhere

Reply:
Good idea!